http2-agent/src/main/java/com/palantir/remoting2/http2/Http2Agent.java

```diff
@@ -7,11 +7,15 @@
 import com.ea.agentloader.AgentLoader;
 import java.lang.instrument.Instrumentation;
 import org.mortbay.jetty.alpn.agent.Premain;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /** A very simple wrapper around the jetty-alpn-agent for dynamic loading. */
 public final class Http2Agent {
     private Http2Agent() {}
 
+    private static final Logger logger = LoggerFactory.getLogger(Http2Agent.class);
```

Comment:
yes, I've been thinking about writing an errorprone check but haven't gotten around to it.